src/DoctrineDBALAdapter.php

```diff
@@ -78,23 +79,23 @@ public function fileExists(string $path): bool
 
     public function directoryExists(string $path): bool
     {
-        return $this->exists($this->prefixer->prefixPath($path), self::TYPE_DIR);
+        return $this->exists($this->prefixer->prefixDirectoryPath($path), self::TYPE_DIR);
```

Comment:
Should not be changed. Directory path in DB does not contain trailing slash. The type column stores the info.